.github/workflows/main.yml

```diff
@@ -10,6 +10,7 @@ env:
   CACHE_REPO: redkubes/otomi-core
   REPO: otomi/core
   GIT_USER: redkubesbot
+  DIGITALOCEAN_PROJECT: 888d7eee-cd33-4f22-8499-274c393f9b80
```

Comment:
Suggestion to remove hardcoded values and use secrets instead.

Reply:
Done.